packages/menu/stories/menu-item.stories.ts

```diff
@@ -41,18 +41,24 @@ export const noWrap = (): TemplateResult => {
 
 export const valueSlot = (): TemplateResult => {
     return html`
+        <style>
+            kbd {
+                font-family: var(--spectrum-alias-body-text-font-family);
+                white-space: nowrap;
+            }
+        </style>
         <sp-menu style="width: 150px;" selectable>
             <sp-menu-item>
                 Save
-                <kbd slot="value">⌘S</kbd>
+                <kbd slot="value">⌘​S</kbd>
             </sp-menu-item>
             <sp-menu-item selected>
                 Save As...
-                <kbd slot="value">⇧⌘S</kbd>
+                <kbd slot="value">⇧​⌘​S</kbd>
             </sp-menu-item>
             <sp-menu-item disabled>
                 Save All
-                <kbd slot="value">⌥⌘S</kbd>
+                <kbd slot="value">⌥​⌘​S</kbd>
```

Comment:
Curious if you know why this changed (text editor differences?)

Reply:
I wish I could explain...

It's even hard to see what the actual change here is. Between the characters in each `<kbd>` element I've added a `zero width space` character. The theory is that it prevent ligatures. as I was getting different sizes of the `⌘` character in the first two menu items than in the third one, which was really weird:

![image](https://user-images.githubusercontent.com/1156657/126660404-34ff3c75-25e0-49f1-9071-bcf954aad788.png)

I made the related CSS changes after the fact, so I'm running a couple branches to make sure. One without the zero width spaces, but the CSS changes, to see if that's enough. Namely thinking that it'll be hard to know you removed the zero width space if you edit this file. And, another with no zero width spaces and `font-variant-ligatures: none;` added to be more specific about it. Then I'll know for sure, hopefully, and can make a note in the story file.

Reply:
If you do need to keep the zero-width spaces, I'd definitely add a comment to indicate that's expected/there.

Reply:
But I hope they're not 😮